New package: LambertsProblem v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: LambertsProblem
- Repository: https://github.com/Burtony3/LambertsProblem.jl
- Created by: @github-actions[bot]
- Version: v0.1.0
- Commit: 6e753fd37d49ee06abb07f59a8cd8af58b9acbf1
- Reviewed by: @github-actions[bot]
- Reference: https://github.com/Burtony3/LambertsProblem.jl/commit/6e753fd37d49ee06abb07f59a8cd8af58b9acbf1#commitcomment-158444865
- Description: Array of Lambert Solvers Writen in Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd674f8710f4373b9ec2fbf241d55fedb09d093f43d1291591b7c6cc23ea0f7aac1eb38837d272c37443ebff5cdd18160e4479024239a27554443d0fd7cc77442fc7b211e37a2703639fb76ef2444657f8027ed2f5751e02f0f3e9b8c3d172e829574a834d114a93f8bca6e2a45562d021edd9eeafb0cf3495bfe4f3ed9b19e32799a100395f93cf759a65f33965d2dc816d42f04456b1dd177903357a4cafbab2d4f6754635135d46c14064fd9fb4b2f6d -->

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though. In any case, make sure the links from the docs-badges in the README work (if they exist, which they currently do).

JuliaRegistrator committed:
New package: LambertsProblem v0.1.0

UUID: bfcf449f-4a4c-42e3-91d7-977e7d435437
Repo: https://github.com/Burtony3/LambertsProblem.jl.git
Tree: bdd0a1ba05014ff73a8a5a47d155951aa2bf94e8

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

Burtony3 commented:
[merge approved]
